apiary.apib

```diff
@@ -1037,6 +1104,135 @@ Available types are:
+
+## InvestmentDetail (object)
+
+- id: `628265` (number, required) - Id of investment
+- loan (object, required)
+    - id: `56321` (number, required) - Id of loan
+    - activeLoanOrdinal: `1` (number, required) - Number of *other* active loans of the same borrower
+    - contractNo: `1007376013` (string, required) - Immutable number identifying loan across Zonky platform
+    - title: `Restrukturalizace úvěrů` (string, required) - Title or name of loan
+    - story: `Dlužím, kam se podívám, a chci to splatit.` (string, required) - Loan story
+    - annuity: `1400` (number, required) - annuity (monthly repayment)
+    - labels: [`VERIFIED_INCOME`] (array[enum], fixed-type) - additional information about loan status
+        - Members
+            - `VERIFIED_BORROWER`
+            - `VERIFIED_INCOME`
+            - `CURRENTLY_INSURED`
+            - `ADDITIONALLY_INSURED`
+            - `BLACK_FRIDAY`
+            - `COVID_19_REQUEST_WAITING`
+            - `COVID_19_POSTPONEMENT_PROCESSED`
+    - borrower (object, required)
+        - id: `125646` (number, required) - borrower internal id
+        - primaryIncomeType: `EMPLOYMENT` (enum, required) - borrower primary income type
+            - Members
+                - `EMPLOYMENT`
+                - `SELF_EMPLOYMENT`
+                - `MATERNITY_LEAVE`
+                - `ENTREPRENEUR`
+                - `STUDENT`
+                - `PENSION`
+                - `LIBERAL_PROFESSION`
+                - `UNEMPLOYED`
+                - `OTHER`
+        - region: `KARLOVARSKY` (enum, required) - borrower region
+            - Members
+                - `HLAVNI_MESTO_PRAHA`
+                - `STREDOCESKY`
+                - `JIHOCESKY`
+                - `PLZENSKY`
+                - `KARLOVARSKY`
+                - `USTECKY`
+                - `LIBERECKY`
+                - `KRALOVEHRADECKY`
+                - `PARDUBICKY`
+                - `VYSOCINA`
+                - `JIHOMORAVSKY`
+                - `OLOMOUCKY`
+                - `MORAVSKOSLEZSKY`
+                - `ZLINSKY`
+    - healthStats (object, required)
+        - paidInstalments: `2` (number, required) - Number of already paid instalments
+        - longestDaysDue: `64` (number, required) - Maximum number of days after a due date of the oldest unpaid instalment in whole history of loan
+        - daysSinceLastInDue: `128` (number, required) - Number of days since last due instalment has been repaid
+        - instalmentsCurrentlyInDue: `1` (number, required) - Number of instalments that are currently in due
+        - loanHealthInfo (enum, required)
+            - Members
+                - `CURRENTLY_IN_DUE` - Last payment is not paid on time
+                - `HISTORICALLY_IN_DUE` - There is delayed payment in past but currently borrower is paying on time
+                - `HEALTHY` - There is no delayed payment during whole time
+- smpSellInfo (object, required)
```

Comment:
I have brought this up in the mailing list earlier, but let me reiterate: having `smpSellInfo` only available on a separate endpoint forces third-party apps to hit Zonky API too often, causing `HTTP 429 Too Many Requests`. The same goes for `healthStats`.

This PR suggests that the API is still being evolved. Perhaps this is something that could be fixed?
If the API intends to limit how many API requests the users can make, then it makes sense for the API to be designed to limit the amount of requests necessary.

Reply:
I'm aware of your concerns and also the email discussion. This PR describes current situation and endpoints, that can be used right now. It was designed primary for our usage and we are doing more improvements for public usage. Unfortunately these additional features are blocked by some other tasks we need to finish first and takes longer that we expected. I hope that we will be able to move requested fields from detail to list endpoint before we remove the old endpoints